### PR TITLE
Implemented input component

### PR DESCRIPTION
**Pull request by mmsbrggr**
| Q | A
| --- | ---
| New feature? | yes
| BC breaks? | no
| Deprecations? | no
| License | MIT

#### What's in this PR?

A general input component which is able to display an icon on the left side of it.

#### Why?

The input component is a central element in sulu.

#### Example Usage

~~~javascript
const onChange = (value) => {console.log(value)};
<Input onChange={onChange} />
~~~

### CODE REVIEW

**Comment by mmsbrggr** [comment]
@danrot This PR can be reviewed. :)

File: src/Sulu/Bundle/AdminBundle/Resources/js/components/Input/Input.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+};
+
+export default class Input extends React.PureComponent<Props> {
+    static defaultProps = {
+        type: 'text',
+    };
+
+    handleChange = (event: SyntheticEvent<HTMLInputElement>) => {
+        if (typeof event.target.value === 'string') {
+            this.props.onChange(event.target.value);
```
Comment (danrot) [inline src/Sulu/Bundle/AdminBundle/Resources/js/components/Input/Input.js]: `event.currentTarget`

File: src/Sulu/Bundle/AdminBundle/Resources/js/components/Input/Input.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        }
+    };
+
+    render() {
+        return (
+            <label className={inputStyles.input}>
```
Comment (danrot) [inline src/Sulu/Bundle/AdminBundle/Resources/js/components/Input/Input.js]: Why `.input`? Isn't `.label` more correct?

Comment (mmsbrggr) [inline src/Sulu/Bundle/AdminBundle/Resources/js/components/Input/Input.js]: It represents the whole input component

**Comment by mmsbrggr** [comment]
@danrot The issue in IE11 is fixed.

**Pull request merged**